README.md

```diff
@@ -1,2 +1,50 @@
-# http-method-override
+# http-method-override [![Build Status](https://travis-ci.org/rstgroup/http-method-override.svg?branch=master)](https://travis-ci.org/rstgroup/http-method-override)
 HTTP method override service
+
+Library allow to override method by header. Why? Because some clients cannot send custom method. It can be used as
+PSR-7 middeware or Zend Framework 2 module.
+
+## Installation
+
+```json
+{
+    "require": {
+        "rstgroup/http-method-override": "dev-master"
+    }
+}
+```
+
+You need to configure how it's possible to override methods. To do that you need to create/modify `config` service in
```

Comment:
why is config quoted? it look's like i have to create 'config' entry in my config